### PR TITLE
Assign Nuage event types to timeline categories

### PR DESCRIPTION
**Pull request by tadeboro**
This will make Nuage events visible in "Monitoring -> Timelines" view.

This pull request depends on https://github.com/ManageIQ/manageiq-ui-classic/pull/4375 and is replacing #80.

@miq-bot add_label wip

/cc @miha-plesko @gberginc

### CODE REVIEW

**Comment by miha-plesko** [comment]
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1562414

File: config/settings.yml [hunk 1]
```diff
@@ -4,6 +4,23 @@
...
+        :addition:
+          :critical_regex:
+          - ^nuage_.+_create$
+        :deletion:
+          :critical_regex:
+          - ^nuage_.+_delete$
+        :update:
+          :critical_regex:
+          - ^nuage_.+_update$
+          :detail:
```
Comment (miha-plesko) [inline config/settings.yml]: Please just remove `:detail:` category. We want all events to land in `:critical:` at this point and will decide which ones to degradate to `:detail:` later.

File: config/settings.yml [hunk 2]
```diff
@@ -4,6 +4,23 @@
...
+          - nuage_vsc_update
+          - nuage_vrs_update
+        :status: # alarms
+          :critical:
```
Comment (miha-plesko) [inline config/settings.yml]: This has changed a little, please remove the `:critical:` category and simply use

```yaml
:critical_regex:
- ^nuage_alarm_.*$
```

File: config/settings.yml (hunk 1 above)
Comment (tadeboro) [reply config/settings.yml]: Will do. I simply converted the code from the older pull request.

File: config/settings.yml (hunk 2 above)
Comment (tadeboro) [reply config/settings.yml]: Consider it done.

**Comment by tadeboro** [comment]
The required work has been merged into core and ui, so this can now be tested against latest master branches of those repositories.

@miq-bot remove_label wip
@miq-bot assign @miha-plesko

**Comment by miha-plesko** [comment]
Kicking Travis now that log directory PR is merged

**Comment by miha-plesko** [comment]
Tested and it works like a charm, thanks @tadeboro!

**Comment by bronaghs** [comment]
@miha-plesko  - Can you add a screenshot?

**Comment by miha-plesko** [comment]
@bronaghs I put some Nuage provider screenshots to this issue here, Event Timelines are also there: https://github.com/ManageIQ/manageiq-providers-nuage/issues/140#issuecomment-416866018

Please let me know if you need some more.

**Pull request merged**